pkg/http/core/kubernetes/deploy.go

```diff
@@ -302,3 +311,31 @@ func handleVersionedManifest(kubeClient kube.Client, u *unstructured.Unstructure
 
 	return nil
 }
+
+func handleUseSourceCapacity(kubeClient kube.Client, u *unstructured.Unstructured, namespaceOverride string) error {
+	current, err := kubeClient.Get(u.GetKind(), u.GetName(), namespaceOverride)
+	if err != nil {
+		if errors.IsNotFound(err) {
```

Comment:
Does this actually work? When I tried to use `errors.IsNotFound` it was not returning true even though the error was defined by the Kubernetes package. See here: https://github.com/homedepot/go-clouddriver/blob/master/pkg/http/core/task.go#L81.

Reply:
Hmm, i had forgotten I had seen this, otherwise I would have done the same.  The comment refers to a problem with testing, but I got that to work by using the `NewNotFound` function from the k8s package

>		// I tried to use `errors.IsNotFound(err)` here to check
>		// if the error was a not found error, but was unable to get the
>		// test to work in doing this, so for now we are just checking
>		// the string suffix.

It seems to work in practice as well, before I added check for not found, the Spinnaker pipeline was failing on the first deployment of a resource.  After I added the check, the pipeline succeeded

<img width="1021" alt="image" src="https://user-images.githubusercontent.com/16955707/134563750-b7c8702f-586b-4157-b572-470307d3c791.png">